feature-libs/product-configurator/rulebased/core/state/actions/configurator.action.ts

```diff
@@ -121,7 +126,20 @@ export class ReadConfiguration extends StateUtils.EntityLoadAction {
 export class ReadConfigurationFail extends StateUtils.EntityFailAction {
   readonly type = READ_CONFIGURATION_FAIL;
 
-  constructor(public payload: { ownerKey: string; error: ErrorActionType }) {
+  /**
+   * @deprecated Please use the `error` parameter with a non-null, non-undefined value.
+   *             Support for `null` or `undefined` will be removed in future versions,
+   *             along with the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
+   */
+  constructor(payload: { ownerKey: string; error: null | undefined });
```

Comment:
`query:` Is the order od constructor's overriding proper? Shouldn't the new, not deprecated definition, be first?